src/main/java/org/springframework/data/couchbase/core/ReactiveFindByQueryOperation.java

```diff
@@ -91,6 +91,15 @@
 		 */
 		TerminatingFindByQuery<T> matching(Query query);
 
+		/**
+		 * Set the filter for the query to be used.
+		 *
+		 * @param queryString must not be {@literal null}.
+		 * @return TerminatingQuery to invoke the actual execution
+		 * @throws IllegalArgumentException if query is {@literal null}.
+		 */
+		TerminatingFindByQuery<T> raw(String queryString);
```

Comment:
String-queries should be a subtype of `Query` so we don't need to update the API for introducing another set of methods accepting a query.

Reply:
done

Reply:
this still needs to be removed, still int he latest push?

Reply:
Yes. Somehow I undid this locally.  Will be in amend.